main/main.cpp

```diff
@@ -1134,6 +1139,19 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 				OS::get_singleton()->print("Missing fixed-fps argument, aborting.\n");
 				goto error;
 			}
+		} else if (I->get() == "--write-movie") {
+			if (I->next()) {
+				write_movie_path = I->next()->get();
+				N = I->next()->next();
+				if (fixed_fps == -1) {
+					fixed_fps = 60;
```

Comment:
I'm not sure if implicitly falling back to 60 FPS is a good idea. I would at least print a message in this case to warn the user that no fixed FPS was specified on the command line, and tell that the default value of 60 will be used.

Reply:
This is only really for running from command line because it needs _something_. The actual writer has a project setting option and it always passed the proper fixed fps value.